docs/PerformAudioPassThru/index.md

```diff
@@ -1,9 +1,6 @@
 ## PerformAudioPassThru
 
-`PerformAudioPassThru` feeds you with audio data through the vehicle’s
-microphone. The audio data could be used in cloud-based and on-line voice
-recognition to achieve dynamic and diversified user interaction, like POI search, information query. Or even record the voice when driver is singing.
-The audio data is in form of PCM, and its sampling rate, bit width, timeout can be set in the `PerformAudioPassThru` request per your requirement. The headunit transmits its supported parameters via the `registerAppInterface` response. 16bit/16kHz is a widely supported configuration, which delivers good audio quality.
+`PerformAudioPassThru` feeds you with audio data through the vehicle’s microphone. The audio data could be used in cloud-based and on-line voice recognition to achieve dynamic and diversified user interaction, like POI search, information query. Or even record the voice when driver is singing. The audio data is in form of PCM, and its sampling rate, bit width, timeout can be set in the `PerformAudioPassThru` request per your requirement. The headunit transmits its supported parameters via the `registerAppInterface` response. 16bit/16kHz is a widely supported configuration, which delivers good audio quality.
```

Comment:
* The `PerformAudioPassThru`  rpc feeds you audio data...
* ...like POI search, querying for information or even recording the driver while they are singing. (I'm not sure what POI is?)
* The audio data returned is formatted using the uncompressed PCM audio file format. 
* I think SDL only supports PCM/16bit/16kHz currently...